docs/README.md

```diff
@@ -33,7 +33,7 @@ Credentials must be provided by
 
```

Comment:
I think we can leave the README at the top level - standard for most projects and tox is complaining it can't find it

Reply:
No probs, I'll change that back now.